tests/test_modeling_tf_common.py

```diff
@@ -270,7 +270,7 @@ def test_saved_model_with_attentions_output(self):
                 else:
                     output = outputs["attentions"]
 
-                self.assertEqual(len(output), num_out)
+                self.assertEqual(len(outputs), num_out)
```

Comment:
I'm confused about what this is testing. `num_out` is somehow the number of inputs (`num_out = len(model(class_inputs_dict))`)? How does this relate to this here to the number of expected outputs? cc @jplu